Use system-openssl in Makefile.
Use /usr/bin/wish in gitk.

diff -r 22f72b7084fd Makefile
--- a/Makefile	Mon Dec 18 17:24:53 2006 -0800
+++ b/Makefile	Mon Dec 18 17:28:34 2006 -0800
@@ -334,18 +334,8 @@ ifeq ($(uname_S),Darwin)
 	NEEDS_SSL_WITH_CRYPTO = YesPlease
 	NEEDS_LIBICONV = YesPlease
 	NO_STRLCPY = YesPlease
-	ifndef NO_FINK
-		ifeq ($(shell test -d /sw/lib && echo y),y)
-			BASIC_CFLAGS += -I/sw/include
-			BASIC_LDFLAGS += -L/sw/lib
-		endif
-	endif
-	ifndef NO_DARWIN_PORTS
-		ifeq ($(shell test -d /opt/local/lib && echo y),y)
-			BASIC_CFLAGS += -I/opt/local/include
-			BASIC_LDFLAGS += -L/opt/local/lib
-		endif
-	endif
+	BASIC_CFLAGS += -I%p/lib/system-openssl/include -I%p/include
+	BASIC_LDFLAGS += -L%p/lib/system-openssl/lib -L%p/lib
 endif
 ifeq ($(uname_S),SunOS)
 	NEEDS_SOCKET = YesPlease
@@ -769,7 +759,7 @@ XDIFF_OBJS=xdiff/xdiffi.o xdiff/xprepare
 
 perl/Makefile: perl/Git.pm perl/Makefile.PL GIT-CFLAGS
 	(cd perl && $(PERL_PATH) Makefile.PL \
-		PREFIX='$(prefix_SQ)')
+	PREFIX='$(prefix_SQ)' LIB='%p/lib/perl5' INSTALLSITEMAN3DIR='%p/share/man/man3')
 
 doc:
 	$(MAKE) -C Documentation all
diff -r 22f72b7084fd gitk
--- a/gitk	Mon Dec 18 17:24:53 2006 -0800
+++ b/gitk	Mon Dec 18 17:25:11 2006 -0800
@@ -1,6 +1,6 @@
 #!/bin/sh
 # Tcl ignores the next line -*- tcl -*- \
-exec wish "$0" -- "$@"
+exec /usr/bin/wish "$0" -- "$@"
 
 # Copyright (C) 2005-2006 Paul Mackerras.  All rights reserved.
 # This program is free software; it may be used, copied, modified
